module/applications/actor/api/base-actor-sheet.mjs

```diff
@@ -1028,6 +1127,12 @@ export default class BaseActorSheet extends PrimarySheetMixin(
       this.element.classList.toggle("sidebar-collapsed", sidebarCollapsed);
     }
 
+    // Attunement
+    this._renderAttunement(context, options);
+
+    // Spellbook
+    this._renderSpellbook(context, options);
```

Comment:
Do we want to be rendering either of these for other actor types such as groups or vehicles? Might be better to call these methods in the character and NPC sheets, rather than in the base sheet.

Reply:
Maybe we will need spellbook on groups for group rituals. I'll move it out for now.